src/modules/remove.js

```diff
@@ -0,0 +1,20 @@
+export const removeAllItems = (tasks, container) => {
+  container.innerHTML = '';
+  return [];
+};
+
+export const removeAllIcon = document.querySelector('.refresf-icon');
```

Comment:
- [ ] Please the refresh icon is for refresh the page and not for deleting all the task please kindly fit it 🙃